tst/testbugfix/00068.tst

```diff
@@ -0,0 +1,5 @@
+# 2005/05/03 (BE)
+gap> NumberSmallGroups(5^6);
+684
+gap> NumberSmallGroups(5*7*9*11*13);
+22
```

Comment:
I gues right now the tests are in the exact same order they appeared in in testbugfix.tst. But I wonder if we could try to automatically group them in a bunch of categries. E.g. if `SmallGroup` occurs, then it' likely a test for small groups. (Of course not always, since some tests will involve small groups just as input for another test). Same for `TransitiveGroup`, `PrimitiveGroup`, etc.

Anyway, that's a thought for later, I guess... I am assuming we will want to remove numbered tests which are subsumed by other tests now, and also group some of these tsts in a more logical manner...